doctrine.rst

```diff
@@ -569,26 +571,18 @@ a controller, this is pretty easy. Add the following method to the
         return new Response('Saved new product with id '.$product->getId());
     }
 
-    // you can also receive the $em as an argument
-    public function editAction(EntityManagerInterface $em)
+    // if you have multiple entity managers, use the registry to fetch them
+    public function editAction(ManagerRegistry $doctrine)
     {
-        // ...
+        $em = $doctrine->getManager();
+        $em2 = $doctrine->getManager('other_connection')
```

Comment:
I am not sure if it is common to get the manager for a particular connection. Don't you rather want to get the entity manager that manages a particular entity (i.e. you will want to use `getManagerForClass()` instead)?